router/router.go

```diff
@@ -592,7 +592,7 @@ func (worker *workerT) processDestinationJobs() {
 	})
 
 	for _, destinationJob := range worker.destinationJobs {
-		var attemptedToSendTheJob bool
+		var attemptedToSendTheJob = true
```

Comment:
seems complex and error prone :) how do we test it?

Reply:
Yeah even I was thinking how can we test these cases

Reply:
Was thinking do we need to complicate this logic and probably we should get rid of the variable all together